WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -671,7 +658,7 @@ - (void)showMediaFailedRemovalAlert
 - (void)showFailedMediaBeforeEditAlert
 {
     UIAlertView * failedMediaBeforeEditAlertView = [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"Failed media", @"Title for alert when trying to edit html post with failed media items")
-                                                           message:NSLocalizedString(@"There are media items in this post that failed to upload to the server. Do you want to remove them?", @"Asks the user if they want to remove their failed media from the post before editing the html")
+                                                           message:NSLocalizedString(@"Some media uploads failed. Manual HTML editing  this post will remove them from the post. Edit anyway?", @"Confirms with the user if they manul edit the post html all media that failed to upload will be removed from it.")
```

Comment:
To keep more consistent with https://github.com/wordpress-mobile/WordPress-iOS-Editor/issues/344, please put "Switch anyway?" and "Save anyway?" on own line after a `br`

@“Some image uploads failed. Switching to the HTML view of this post will remove failed media. Switch anyway?", @"Confirms with the user if they manually edit the post html all media that failed to upload will be removed from it.”
> Not Now/Yes

@"Some image uploads failed. This action will remove all failed images from the post. Save anyway?", @"Confirms with the user if they save the post all media that failed to upload will be removed from it."
> Not Now/Yes